raiden/tests/utils/smoketest.py

```diff
@@ -43,36 +46,15 @@
 TEST_PARTNER_ADDRESS = '2' * 40
 TEST_DEPOSIT_AMOUNT = 5
 
+TEST_PRIVKEY = (
+    b'\xad\xd4\xd3\x10\xba\x04$hy\x1d\xd7\xbf\x7fn\xae\x85\xac'
+    b'\xc4\xdd\x14?\xfa\x81\x0e\xf1\x80\x9aj\x11\xf2\xbcD'
+)
+TEST_ACCOUNT_ADDRESS = privatekey_to_address(TEST_PRIVKEY)
+
 RST_DATADIR = tempfile.mkdtemp()
 os.environ['RST_DATADIR'] = RST_DATADIR
 
-GENESIS_PATH = Template('$RST_DATADIR/genesis.json').substitute(os.environ)
-
-# Environment variables in the command for datadir and ports allow customization.
-# We use DEFAULT_ETH_COMMAND, unless '$RST_ETH_COMMAND' is defined (all environment variables
-# specific to the raiden smoke test are prefixed 'RST_' for Raiden Smoke Test).
-# For customization, set the environment variable to fit your client like this:
-# RST_ETH_COMMAND="ethereum --rpc-port \$RST_RPC_PORT \
-#        --data-dir \$RST_DATADIR" raiden smoketest
-# FIXME: this does not work: the `init` phase is not customizable (gh issue #758)
-DEFAULT_ETH_COMMAND = """
-$RST_GETH_BINARY
-    --nodiscover
-    --nat none
-    --port 0
-    --ipcdisable
-    --rpc
-    --rpcaddr 127.0.0.1
-    --rpcapi eth,net,web3,personal
```

Comment:
this is a bit different from what we have in `raiden.tests.utils.geth`, but the smoketest runs fine.